addons/website_slides_forum/views/forum_views.xml

```diff
@@ -22,6 +22,20 @@
         </field>
     </record>
 
+      <record id="view_forum_forum_list" model="ir.ui.view">
```

Comment:
Id should be ``forum_forum_view_list`` (even if base view does not follow guidelines, inherited views should)